src/components/reusuables/cards/ServiceCard.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import { StaticImage } from "gatsby-plugin-image"
+import { Link } from "gatsby"
+
+const ServiceCard = (props) => {
```

Comment:
Change props to the prop name. Example, (props)-> ({heading,subtitle})

Reply:
Changed props to prop name